package/python3-img-proof.spec

```diff
@@ -25,6 +25,7 @@ License:        GPL-3.0-or-later
 Group:          Development/Languages/Python
 URL:            https://github.com/SUSE-Enceladus/img-proof
 Source:         https://files.pythonhosted.org/packages/source/i/img-proof/img-proof-%{version}.tar.gz
+BuildRequires:  python3-rpm-macros
```

Comment:
Please use `python-rpm-macros`. `python3-rpm-macros` is just a `Provides` and as such if someone searches for `python3-rpm-macros` in the Build Service nothing will show up which will lead to an unnecessary goose chase.

Reply:
Rebased